### PR TITLE
added spanish bible text and class

### PR DESCRIPTION
**Pull request by sanggwonjang**

### CODE REVIEW

File: lib/services/bible/spanish_bible.dart [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+    // First check through the abbreviations.
+    var shortNames = shortenedBookNames;
+    for (int i = 0; i < shortNames.length; i++) {
+      if (shortNames[i].toLowerCase() == book.toLowerCase()) return i;
+    }
+    return super.getBookIndex(book);
+  }
+
+  static const shortBookNames = [
```
Comment (hyeonwooshim) [inline lib/services/bible/spanish_bible.dart:29]: let's just do `shortBookNames = fullBookNames`.
The search part will take care of the spaces.

**Comment by hyeonwooshim** [comment]
it would be nice to have a screenshot :D

File: lib/services/bible/spanish_bible.dart (hunk 1 above)
Comment (sanggwonjang) [reply lib/services/bible/spanish_bible.dart:29]: Changing to `shortBookNames = fullBookNames` seems to cause problem when trying to load Bible

Comment (hyeonwooshim) [reply lib/services/bible/spanish_bible.dart:29]: Oh I see. The problem occurs in `getBookIndex` right?
How about we generate `shortBookNames` by removing spaces from `fullBookNames`?
Isn't that what these values are?

**Pull request merged**